### DISCUSSION TITLE
Stuck in AI Tree by Rotate to face BB entry

### DISCUSSION THREAD

**Discussion by dream4java** [Sixze/ALS-Refactored#366] [Q&A]
Stuck in Rotate to face BB entry

**Comment by adamfulop** [comment]
This issue still occurs, the root of the cause is that under a certain threshold the character just looks in the direction, does not turn to it, leaving rotate to face BB entry node infinite hanging.

If I reduce the threshold to like ~1-5 degrees with the turn in place or rotation trigger (depending on the view state), the character starts infinitely oscillating left-right, can't stop at the desired rotation.

Is there any fix for this? It would be a rather crucial feature.
